meta-resin-common/recipes-support/resin-device-uuid/resin-device-uuid.bb

```diff
@@ -0,0 +1,40 @@
+DESCRIPTION = "resin device uuid"
+SECTION = "console/utils"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${RESIN_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+PR = "r2"
+
+SRC_URI = " \
+    file://resin-device-uuid \
+    file://resin-device-uuid.service \
+    "
+S = "${WORKDIR}"
+
+inherit allarch systemd
+
+RDEPENDS_${PN} = " \
+    bash \
+    curl \
+    jq \
+    resin-conf \
+    openssl \
+    docker-resin-supervisor-disk \
```

Comment:
This will be included, but it will also bring docker-resin-supervisor-disk. Is this not intended?